core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/score/director/ScoreDirectorFactoryFactory.java

```diff
@@ -149,6 +151,10 @@ protected AbstractScoreDirectorFactory<Solution_, Score_> decideMultipleScoreDir
                 + "easyScoreCalculatorClass or an incrementalScoreCalculatorClass.");
     }
 
+    public static void failForMissingConstraintStreams(ConstraintStreamImplType constraintStreamImplType) {
+
```

Comment:
@triceo Is this empty method intentional?

Reply:
It's unused as far as I can tell.

Reply:
Good point. Leftover from an earlier version. Will delete.

Reply:
Thanks :)